### PR TITLE
OHRM5X-1508: Develop localization - employee name translate composable

### PR DESCRIPTION
**Pull request by devishke-orange**
PR contains:
- OHRM5X-1508: [Broken] [Timesheets] Purged Employee tag is not displayed properly.
- OHRM5X-1468: reopen issue
- OHRM5X-1517: [Review Evaluation] Review summary employee job title is getting updated according to the current job title

Other:
- Change purged employee name tag behaviour in Project List/Add/Edit screens
- Improved code coverage for performance

### CODE REVIEW

File: src/client/src/core/util/composable/useEmployeeNameTranslate.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  lastName: string;
+  terminationId: number | null;
+};
+
+export default function useEmployeeNameTranslate() {
+  const {$t} = usei18n();
+
+  const translateEmployeeName = (
+    employee: Employee,
+    includeMiddle?: boolean,
```
Comment (RajithaKumara) [inline src/client/src/core/util/composable/useEmployeeNameTranslate.ts:33]: :ok_hand:

File: src/plugins/orangehrmPerformancePlugin/test/Entity/PerformanceReviewTest.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+class PerformanceReviewTest extends EntityTestCase
+{
+    protected function setUp(): void
+    {
+        TestDataService::truncateSpecificTables([Employee::class]);
+        TestDataService::truncateSpecificTables([JobTitle::class]);
+        TestDataService::truncateSpecificTables([Subunit::class]);
+        TestDataService::truncateSpecificTables([PerformanceReview::class]);
+        TestDataService::truncateSpecificTables([Reviewer::class]);
```
Comment (RajithaKumara) [inline src/plugins/orangehrmPerformancePlugin/test/Entity/PerformanceReviewTest.php:40]:
```suggestion
        TestDataService::truncateSpecificTables([Employee::class, JobTitle::class, Subunit::class, PerformanceReview::class, Reviewer::class]);
```
:thinking:

File: src/plugins/orangehrmPerformancePlugin/test/Entity/PerformanceTrackerLogTest.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+ */
+class PerformanceTrackerLogTest extends EntityTestCase
+{
+    protected function setUp(): void
+    {
+        TestDataService::truncateSpecificTables([Employee::class]);
+        TestDataService::truncateSpecificTables([User::class]);
+        TestDataService::truncateSpecificTables([UserRole::class]);
+        TestDataService::truncateSpecificTables([PerformanceTracker::class]);
+        TestDataService::truncateSpecificTables([PerformanceTrackerLog::class]);
```
Comment (RajithaKumara) [inline src/plugins/orangehrmPerformancePlugin/test/Entity/PerformanceTrackerLogTest.php:43]: *

File: src/plugins/orangehrmPerformancePlugin/test/Entity/ReviewerTest.php [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+use DateTime;
+use OrangeHRM\Entity\Employee;
+use OrangeHRM\Entity\PerformanceReview;
+use OrangeHRM\Entity\Reviewer;
+use OrangeHRM\Entity\ReviewerGroup;
+use OrangeHRM\Tests\Util\EntityTestCase;
+use OrangeHRM\Tests\Util\TestDataService;
+
+class ReviewerTest extends EntityTestCase
```
Comment (RajithaKumara) [inline src/plugins/orangehrmPerformancePlugin/test/Entity/ReviewerTest.php:30]: add group

File: src/plugins/orangehrmPerformancePlugin/test/Entity/ReviewerRatingTest.php [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+use OrangeHRM\Entity\JobTitle;
+use OrangeHRM\Entity\Kpi;
+use OrangeHRM\Entity\PerformanceReview;
+use OrangeHRM\Entity\Reviewer;
+use OrangeHRM\Entity\ReviewerRating;
+use OrangeHRM\Tests\Util\EntityTestCase;
+use OrangeHRM\Tests\Util\TestDataService;
+
+class ReviewerRatingTest extends EntityTestCase
```
[the remark this answers is not in the document]
Comment (devishke-orange) [reply src/plugins/orangehrmPerformancePlugin/test/Entity/ReviewerRatingTest.php:34]: added

**Review by Super-Chama — APPROVED** [review]

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**